x/bonds/internal/types/msgs_test.go

```diff
@@ -15,7 +15,8 @@ func TestValidateBasicMsgCreateTokenArgumentMissingGivesError(t *testing.T) {
 	err := message.ValidateBasic()
 
 	require.NotNil(t, err)
-	require.Equal(t, CodeArgumentInvalid, err.Code())
+	// TODO: is this necessary?
+	// require.Equal(t, CodeArgumentInvalid, err.Code())
```

Comment:
It is not strictly necessary. But it was just a way to confirm that the error generated is exactly the one expected and not just any other error. Ideally these are uncommented rather than removed. (applies to entire file)

Reply:
Unfortunately now we do not have the codes accessible since they are integrated into the error definition. So in reality I think we should remove them.